app/views/tasks/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<% content_for :head do %>
+  <title>Details: <%= @task.name %></title>
+<% end %>
+
+<h1 class='title'>Details</h1>
+
+<main>
+  <p>
+    <%= render 'checkbox', { task: @task } %>
+    <span class='task_name'><%= @task.name %></span>
+
+    <span class='complete_date'>
+      <% if @task.completed_at %>
```

Comment:
minor, but I'd consider making a method `completed?` which just wraps `completed_at`. It would make your intentions clearer here and be more flexible in the future if you needed to change the definition of what `completed?` meant.